server/src/test/java/org/elasticsearch/cluster/coordination/AtomicRegisterCoordinatorTests.java

```diff
@@ -234,36 +237,48 @@ protected CoordinatorStrategy getCoordinatorStrategy() {
         return new AtomicRegisterCoordinatorStrategy(atomicRegister, sharedStore);
     }
 
-    record HeartBeat(DiscoveryNode leader, long term, long absoluteTimeInMillis) {
+    public record Heartbeat(DiscoveryNode leader, long term, long absoluteTimeInMillis) {
         long timeSinceLastHeartbeatInMillis(long nowInMillis) {
             return nowInMillis - absoluteTimeInMillis;
         }
     }
 
-    static class StoreHeartbeatService implements LeaderHeartbeatService {
+    public static class StoreHeartbeatService implements LeaderHeartbeatService {
+        public static final Setting<TimeValue> HEARTBEAT_FREQUENCY = Setting.timeSetting(
+            "heartbeat_frequency",
```

Comment:
NIT: I think this is interval since it is configured with a time value

Reply:
Aha, I see it was moved as is from another place. Not sure if we should change it at this point